tests/Bot.Builder.Community.Adapters.Tests/ApplicationBuilderExtensionsTests.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Microsoft.AspNetCore.Builder;
+using Moq;
+using Xunit;
+using Microsoft.Extensions.Options;
+
+namespace Bot.Builder.Community.Adapters.Twitter.Tests
+{
+    [Trait("TestCategory", "Twitter")]
+    public class ApplicationBuilderExtensionsTests
+    {
+        private static readonly Mock<IApplicationBuilder> _testApp = new Mock<IApplicationBuilder>();
+        private static readonly Mock<IServiceProvider> _testService = new Mock<IServiceProvider>();
+        private static readonly Mock<IOptions<TwitterOptions>> _testOptions = new Mock<IOptions<TwitterOptions>>();
```

Comment:
Static properties should start with Uppercase.
```suggestion
        private static readonly Mock<IApplicationBuilder> TestApp = new Mock<IApplicationBuilder>();
        private static readonly Mock<IServiceProvider> TestService = new Mock<IServiceProvider>();
        private static readonly Mock<IOptions<TwitterOptions>> TestOptions = new Mock<IOptions<TwitterOptions>>();
```